docs/MO_DG/prepare_model/FP16_Compression.md

```diff
@@ -2,18 +2,16 @@
 
 @sphinxdirective
 
-Model Optimizer by default converts all floating-point weights to ``FP16`` data type. 
+Model Optimizer can convert all floating-point weights to ``FP16`` data type. 
 The resulting IR is called compressed ``FP16`` model. The resulting model will occupy 
 about twice as less space in the file system, but it may have some accuracy drop. 
-For most models, the accuracy drop is negligible. But in case if accuracy drop is 
-significant user can disable compression explicitly.
+For most models, the accuracy drop is negligible.
 
-By default, models are compressed to ``FP16``, but you can disable compression by 
-specifying ``--compress_to_fp16=False``:
+To compress the model, use the `--compress_to_fp16` or `--compress_to_fp16=True` option:
 
 .. code-block:: sh
 
-   mo --input_model INPUT_MODEL --compress_to_fp16=False
+   mo --input_model INPUT_MODEL --compress_to_fp16
```

Comment:
@kblaszczak-intel, please review these documentation changes. Any were else we need to provide adjustments?

Reply:
I've suggested a minor rewording but it does not change the sense. I can't find any other articles that would require adjusting to this issue.